src/main/java/com/minecraftabnormals/abnormals_core/core/mixin/EntityMixin.java

```diff
@@ -28,9 +27,12 @@
 import java.util.Set;
 
 @Mixin(Entity.class)
-public final class EntityMixin implements IDataManager {
+public abstract class EntityMixin implements IDataManager {
 	@Shadow
-	private World level;
+	public World level;
```

Comment:
Keep this private.
```suggestion
	private World level;
```

Reply:
but the field itself is public, Shadows should normally retain the same access level as their targets, regardless whether their intended uses are internal or external to the mixin in question.